app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
```

Comment:
- [ ] As part of this module requirement, _Comments_ model should have a method that **updates the comments counter** for a post. This should be similar to:

```ruby
# app/models/comment.rb
...

def update_comments_counter
  post.increment!(:comments_counter)
end
```